C/Check/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Check"
+version = v"0.15.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/libcheck/check/releases/download/0.15.1/check-0.15.1.tar.gz", "c1cc3d64975c0edd8042ab90d881662f1571278f8ea79d8e3c2cc877dac60001")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd check-0.15.1
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make
+make install
+exit
```

Comment:
```suggestion
install_license COPYING.LESSER
```

Reply:
What is this for?

Reply:
https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Installing-the-license-file

Reply:
Thanks!